### ISSUE TITLE
Allow area probe plots across different resolutions

### ISSUE THREAD

**Issue by djhoese** [ssec/sift#92]
In GitLab by @djhoese on Oct 26, 2015, 14:47

Currently there is no way to do a scatter plot using bands with different resolutions. The best way to do this is probably some nearest neighbor calculation (cKDTree from scipy).

**Comment by djhoese** [comment]
In GitLab by @jgerth on Oct 26, 2015, 15:31

Tim also reported this as an issue. Would it be possible to produce a set of B03 files at 2 km resolution? Otherwise, we cannot compare it to other bands, and unfortunately, the visible band is a fairly popular band to use in comparisons.

**Comment by djhoese** [comment]
In GitLab by @rayg-ssec on Oct 26, 2015, 17:31

Dave is looking at consolidating this with region extraction & masking prior to making histograms/scatter plots.

**Comment by djhoese** [comment]
In GitLab by @tims on Oct 27, 2015, 08:16

As you know, there are also several 1km bands (1, 2 and 4), so I don't think this is just a band 3 issue (but agree that's the most important). Thanks.

**Comment by djhoese** [comment]
In GitLab by @djhoese on Oct 29, 2015, 18:25

This should work properly in 0.5.1 and with all the other changes I made it should seem faster while switching between tabs and/or plotting for different layers. I chose to upsample any lower res data to the higher resolution layer's pixels. It's like nearest neighbor but I'm literally mapping a pixel's mercator X/Y coordinate in one resolution to the index coordinate (array index) in the other resolution.
